config/locales/en.yml

```diff
@@ -70,5 +70,18 @@ en:
     index:
       title: 'Forms'
     edit:
-      title: "Create Your Form"
+      title: "Add a Field"
       preview: "Preview"
+
+  form_elements:
+    shared:
+      label: 'Label'
+      data_type: 'Field type'
+
+  plugins:
+    action:
+      action_description: "Provide a description for your petition"
+      choose_form_template: "Choose another form template"
```

Comment:
yeah that sounds good to me. why was it 'another'? is there something I didn't understand?